tests/perl/tests.bats

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bats
+
+# Copyright 2017 tsuru authors. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+setup() {
+    rm -rf /home/application/current && mkdir /home/application/current
+    chown ubuntu /home/application/current
+    export CURRENT_DIR=/home/application/current
+}
+
+@test "check if carton is intalled" {
+    pushd ${CURRENT_DIR}
+    run carton -v
```

Comment:
In the context of this test, do I have the example app?
As in [python](https://github.com/tsuru/platforms/blob/master/tests/python/tests.bats#L102) I can add a cpanfile and ask for the carton to install, but would need internet access. Is that okay?